sdk/python/pipeline.yaml

```diff
@@ -0,0 +1,170 @@
+apiVersion: argoproj.io/v1alpha1
```

Comment:
This file probably should not be checked in.

Reply:
done